scripts/lint_site.sh

```diff
@@ -95,7 +95,7 @@ check_content() {
         FAILED=1
     fi
 
-    if grep -nrP --include "*.md" -e "\(https://istio.io/(?!v[0-9]\.[0-9]/|archive/)" .; then
+    if grep -nrP --include "*.md" -e "\(https://istio.io/(?!v[0-9]\.[0-9]/|archive|news/)" .; then
```

Comment:
What is this trying to do?

We shouldn't have any content at https://istio.io/news/ any more, which I think is what is being added.

Reply:
Doesn't /news redirect to /latest/news? I can update the regex and reference though if that's preferred

Reply:
please do yes; everything without a /latest/ is a leftover from before we put versioning in the site, rather than having archives on archive.istio.io.   Which I still don't understand why we did, but I assume David had a really good reason :)

Reply:
Gotcha, updated!